display_list/display_list_sampling_options.h

```diff
@@ -63,10 +46,10 @@ inline DlImageSampling ToDl(const SkSamplingOptions& so) {
   if (so.filter == SkFilterMode::kNearest && so.mipmap == SkMipmapMode::kNone) {
```

Comment:
This if statement isn't useful any more, now that the default return value below it is kNearest.